dts/bindings/tach/ite,it8xxx2-tach.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2021 ITE Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: ITE, it8xxx2 Tachometer node
+
+compatible: "ite,it8xxx2-tach"
+
+include: tach.yaml
+
+properties:
+    reg:
+        required: true
+
+    dvs_bit:
+      type: int
+      required: true
+      description: tachometer data valid bit of tswctlr register
+
+    chsel_bit:
+      type: int
+      required: true
+      description: tachometer data valid status bit of tswctlr register
+
+    pinctrl-0:
+        type: phandles
+        required: true
+        description: configuration of tachometer pinmux controller
+
+    channel:
+      type: int
+      required: true
+      enum:
+            - 0 #TACH_CHANNEL_A
+            - 1 #TACH_CHANNEL_B
```

Comment:
Putting the 0=A, 1=B information in the description rather than a comment would be better, so it shows up in https://docs.zephyrproject.org/latest/reference/devicetree/bindings.html

Not a blocking comment

Reply:
Done